src/ape_pm/compiler.py

```diff
@@ -39,6 +39,10 @@ def compile(
             ):
                 # Raw contract type JSON
                 contract_type_data = data
+                if "deployedBytecode" in data:
+                    contract_type_data["runtimeBytecode"] = {"bytecode": data["deployedBytecode"]}
```

Comment:
If there is also `runtimeBytecode` already, do we still wish to override it?

Reply:
I would say so, just for the sake of completeness (as the vyper compiler also adds that)

Reply:
Ohhh I get it now!

Reply:
The issue is that these are different JSONs. This `if` block is assuming `ContractType.json()`-like structure, which is like step above the raw compiler output.